src/java/org/apache/cassandra/index/SecondaryIndexManager.java

```diff
@@ -847,8 +847,6 @@ private synchronized void markIndexFailed(Index index, boolean isInitialBuild)
 
             if (!index.getSupportedLoadTypeOnFailure(isInitialBuild).supportsReads() && queryableIndexes.remove(indexName))
                 logger.info("Index [{}] became not-queryable because of failed build.", indexName);
-
-            makeIndexNonQueryable(index, Index.Status.BUILD_FAILED);
```

Comment:
This doesn't work as recently we overrode the `isQueryable` for `CassandraIndex`, and `BUILD_FAILED` is a non-queryable index, but `FULL_REBUILD_STARTED` is queryable. So, we would introduce a change of behavior with this call, which we do not want to do.